### ISSUE TITLE
Limited to 25 LORAs

### ISSUE THREAD

**Issue by Kazrik** [ambocclusion/ComfyUI-SDXL-DiscordBot#8]
Hi, I've come to notice today that the bot only supports up to 25 LORA files until some no longer show in the list. It looks like this'd be a limitation of the Discord api but do you know of any workaround?

**Comment by ambocclusion** [comment]
Yeah this is currently because the Discord API only supports 25. I have the ability to add <lora:loraname:1.0> to the prompt similar to A1111. This is not the best workaround but it does allow for any number of loras. You can type /lora in the chat to see a list of all the available loras so at least you can find out what the options are. (sorry for the delayed response)
